flash/image/style_transfer/model.py

```diff
@@ -0,0 +1,174 @@
+from typing import Any, cast, Dict, List, Mapping, NoReturn, Optional, Sequence, Tuple, Type, Union
+
+import torch
+from torch import nn
+from torch.optim.lr_scheduler import _LRScheduler
+
+from flash.core.data.data_source import DefaultDataKeys
+from flash.core.data.process import Serializer
+from flash.core.model import Task
+from flash.core.registry import FlashRegistry
+from flash.core.utilities.imports import _IMAGE_STLYE_TRANSFER
+from flash.image.style_transfer import STYLE_TRANSFER_BACKBONES
+
+if _IMAGE_STLYE_TRANSFER:
+    import pystiche.demo
+    from pystiche import enc, loss, ops
+    from pystiche.image import read_image
+else:
+
+    class enc:
```

Comment:
`CamelCase` class name

Reply:
Following pystiche.

Reply:
This just happens because these classes are actually modules.